library/core/src/cell/once.rs

```diff
@@ -256,7 +259,7 @@ impl<T> OnceCell<T> {
     ///
     /// let mut cell: OnceCell<u32> = OnceCell::new();
     ///
-    /// // Failed initializers do not change the value
+    /// // Failed attempts to initialize the contents of the cell has no effect on the cell
```

Comment:
```suggestion
    /// // Failed attempts to initialize the cell do not change its contents
```

Slightly clearer with flipped wording